docs/user/exercises/programming-exercise-setup.rst

```diff
@@ -116,11 +116,49 @@ Exercise Creation
 
 3. **Update exercise code in repositories**
 
-- **Alternative 1:** Clone the 3 repositories and adapt the code on your local computer in your preferred development environment (e.g. Eclipse)
+- **Alternative 1:** Clone the 3 repositories and adapt the code on your local computer in your preferred development environment (e.g. Eclipse).
 
   - To execute tests, copy the template (or solution) code into a folder **assignment** in the test repository and execute the tests (e.g. using maven clean test)
   - Commit and push your changes |submit|
 
+  - **Notes for Haskell:** In addition to the assignment folder, the executables of the build file expect the solution repository checked out in the **solution** subdirectory of the test folder and also allow for a **template** subdirectory to easily test the template on your local machine.
+    You can use the following script to conveniently checkout an exercise and create the right folder structure:
+
+    .. code-block:: bash
+
+      #!/bin/sh
+      # Arguments:
+      # $1: exercise short name as specified on Artemis
+      # $2: (optional) output folder name
+
+      # base URL to repositories
```

Comment:
```suggestion
      # base URL to repositories (change as needed), e.g.:
```

Reply:
I would add this statement or even more to make sure people understand that this BASE can be changed in case they use any other URL in the version control system

Reply:
Added some comments.